rotkehlchen/chain/ethereum/interfaces/ammswap/ammswap.py

```diff
@@ -725,6 +728,31 @@ def _get_trades(
         self.database.add_amm_swaps(list(all_swaps))
         return self._fetch_trades_from_db(addresses, from_timestamp, to_timestamp)
 
+    def _get_unknown_asset_price_chain(
+        self,
+        unknown_assets: Set[EthereumToken],
+    ) -> AssetToPrice:
+        """Get token price using Uniswap V3 Oracle."""
+        oracle = UniswapV3Oracle(eth_manager=self.ethereum)
+        asset_price: AssetToPrice = {}
+        usdc_asset_identifier = '_ceth_0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'
+        for from_asset in unknown_assets:
+            try:
+                price = oracle.get_price(
+                    Asset(identifier=from_asset.identifier),
+                    Asset(identifier=usdc_asset_identifier),
```

Comment:
Why are you converting to asset here? `EthereumToken` is a subclass of `Asset` already